packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -92,12 +88,11 @@ export class WorkspaceContextUtil {
   }
 
   protected async handleCliConfigChange() {
-    const usernameOrAlias = await this.getAuthUtil().getDefaultUsernameOrAlias(
-      false
-    );
+    await ConfigAggregatorProvider.getInstance().reloadConfigAggregators();
+    const usernameOrAlias = await ConfigUtil.getDefaultUsernameOrAlias();
 
     if (usernameOrAlias) {
-      this._username = await this.getAuthUtil().getUsername(usernameOrAlias);
+      this._username = await ConfigUtil.getUsername();
```

Comment:
This is why the change to resolveValue is needed.  when there is no alias, username is passed in here, and username should come back